src/styles.css

```diff
@@ -8,4 +8,28 @@
 
 button.ui.button {
     opacity: 1 !important;
+}
+
+div.ui.input i {
+    color: #ffffff;
+    opacity: 1 !important;
+}
+
+div.ui.input input, div.ui.input input::placeholder {
+    background-color: #141f3c;
```

Comment:
Background here is still not `transparent`

Reply:
Fixed!

Reply:
You may not be pushing up the changes, because it has not been fixed yet. `#141f3c` should be `transparent`

Reply:
Oh! My bad, It's seems like I forgot to add that file...